podstawowe_operacje/Comparator/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿try
+{
+    Console.WriteLine("Podaj dwie liczby całkowite:");
+    int a;
+    var aSuccess = int.TryParse(Console.ReadLine(), out a);
+    int b;
+    var bSuccess = int.TryParse(Console.ReadLine(), out b);
+
+    if (aSuccess == true && bSuccess == true)
+    {
+        ChooseOperation(a, b);
+    }
+    else
+    {
+        Console.WriteLine($"Zły format liczb");
+        Environment.Exit(0);
```

Comment:
FIX: Staraj się nie używać tej metody. Lepiej:
```suggestion
        return;
```

Reply:
[915aa22](https://github.com/broniuu/Wst-pDoProgramowania/pull/1/commits/915aa22c8e6ca87d6e3393c1875a8ba562dd150b)

Reply:
OK